src/main/java/org/mitre/inferno/rest/ValidatorEndpoint.java

```diff
@@ -39,11 +39,30 @@ public static ValidatorEndpoint getInstance(Validator validator) {
    * Creates the API routes for receiving and processing requests for the validation service.
    */
   private void createRoutes() {
+
+    post("/*/%24validate",
+    (req, res) -> {
+      res.type("application/fhir+json");
+      return validateResource(req.bodyAsBytes(), req.queryParams("profile"));
+    });
+
+    post("/*/*/%24validate",
```

Comment:
```suggestion
    post("/*/*/%24validate",
```
Did you test whether one wildcard captures both `/[Resource]/[id]` and `/[Resource]` ?

Reply:
Also, when adding API endpoints, make sure you update the API documentation.